apps/hosts/migrations/0004_auto_20190111_0007.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you please name the migrations as `Add_blank_in_host_team_URL.py`
You can name the migrations while generating them using
`python manage.py makemigrations --name=Add_blank_in_host_team_URL --settings=settings.dev`

Reply:
@RishabhJain2018 done.